photosynthesis_metrics/tv.py

```diff
@@ -0,0 +1,153 @@
+r"""Implemetation of Total Variation metric, based on article
```

Comment:
This file seems like the exact copy of the file with this name from the previous pull request. Did you include this on purpose? If so, what is your motivation?

Reply:
Discussed during weekly meeting.